yarpl/include/yarpl/observable/ObservableOperator.h

```diff
@@ -18,7 +18,7 @@ namespace observable {
  * pipelines
  * can be built: a Observable heading a sequence of Operators.
  */
-template <typename U, typename D>
+template <typename U, typename D, typename ThisOperatorT>
```

Comment:
I suggest renaming the template name from `ThisOperatorT` to smt shorter. All other templates are like U, D, F..